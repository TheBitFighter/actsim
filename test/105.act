defproc test()
{
   int<8> w;
   chp {
      w := 0x42;
      log ("%bw is currently: ", w);
      w{0} := 1;
      log ("%bw is currently: ", w);
      w{4..2} := 0b101;
      log ("%bw is currently: ", w);
      skip
   }
}

test t;
