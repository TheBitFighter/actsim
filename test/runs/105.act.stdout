[                  10] <>  w is currently: 01000010
[                  20] <>  w is currently: 01000011
[                  30] <>  w is currently: 01010111
